deps/build.jl

```diff
@@ -14,7 +14,17 @@ if is_windows()
     end
 end
 
-libhttp_parser = library_dependency("libhttp_parser", aliases=aliases)
+function validate_httpparser(name,handle)
+    try
+        p = Libdl.dlsym(handle, :http_parser_url_init)
```

Comment:
then this won't be specific enough for anyone who installed the package on master

this kind of detail should be written out in comments